nvflare/edge/tools/edge_recipe.py

```diff
@@ -49,6 +69,21 @@ def __init__(
 
 
 class DeviceManagerConfig:
+    """Configuration class for the device manager in federated learning.
+
+    This class configures how the device manager selects and manages devices
+    for participation in federated learning workflow.
+
+    Attributes:
+        device_selection_size: Number of devices to select for each training round.
+            Default: 100
+        min_hole_to_fill: Minimum number of model updates to wait for before
+            sampling the next batch of devices. Default: 1 (immediately dispatch the current global model)
+        device_reuse: Whether to allow devices to participate in multiple rounds.
```

Comment:
if devide_reuse is False, then this device will only participate ONCE in a round for the whole training process?